build/v1/types.go

```diff
@@ -963,6 +963,11 @@ const (
 type BuildConfigStatus struct {
 	// lastVersion is used to inform about number of last triggered build.
 	LastVersion int64 `json:"lastVersion" protobuf:"varint,1,opt,name=lastVersion"`
+
+	// imageChangeTriggersState is used to capture the runtime state of any ImageChangeTrigers,
+	// including the lastTriggeredImageID and paused fields.  The same type is reused for both spec and status to help
+	// the ImageChangeController correlate between the two.
+	ImageChangeTriggersState []ImageChangeTrigger `json:"imageChangeTriggersState,omitempty" protobuf:"bytes,2,rep,name=imageChangeTriggersState"`
```

Comment:
per discussion and resulting investigation, I propose dropping "State" from the field name:

```suggestion
	// imageChangeTriggers captures the image references used by the build's imageChangeTriggers
	// +optional
	ImageChangeTriggers []ImageChangeTrigger `json:"imageChangeTriggers,omitempty" protobuf:"bytes,2,rep,name=imageChangeTriggers"`
```